docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -139,6 +139,29 @@ The body of the request to /projects/$ProjectName will be an entity of the form:
 }
 ```
 
+```
+GET /projects/$ProjectName/images/
+```
+
+Will return a JSON object of the form
+
+```js
+{
+  "Images" : [
+    {
+      "candidate": "123456",
+      "visit": "V1",
+      "scan_type": Acquisition protocol ex :"t2",
```

Comment:
the reason for using "js" as the type for the block instead of json elsewhere is so that you can use comments. So you can do smething like `"scan_type": "t2", /* acquisition protocol */` and have it highlighted properly in the markdown.

The capitalization of this is also inconsistent with the rest of the endpoints (or at least the ones showing up as context in this diff) where the first letter is capitalized, and CapitalizedWordCase is used, not underscores.